src/domain/customers/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { RouteComponentProps, Router } from "@reach/router"
+import React from "react"
+import PageDescription from "../../components/atoms/page-description"
+import BodyCard from "../../components/organisms/body-card"
+import CustomerTable from "../../components/templates/customer-table"
+import Details from "./details"
+
+const CustomerIndex: React.FC<RouteComponentProps> = () => {
+  return (
+    <div className="flex flex-col grow h-full">
+      <PageDescription
+        title={"Customers"}
+        subtitle="Manage the Customers of your store"
+      />
+      <div className="w-full flex flex-col grow">
+        <BodyCard
+          title="Overview"
+          subtitle="An overview of all of your existing customers"
+          className="mb-0"
+        >
+          <div className="flex grow  flex-col pt-2 mt-large">
+            <CustomerTable />
+          </div>
+        </BodyCard>
+      </div>
+    </div>
+  )
+}
+
+const Customers = () => {
+  return (
+    <Router>
```

Comment:
```suggestion
    <Router className="h-full" >
```

Reply:
Adding this class results in the following table view. I ran it by Ludvig and he would rather have the "shrinked" table so he's gonna update the table designs accordingly 👍 
![image](https://user-images.githubusercontent.com/59018053/151007094-15f984cb-3f2d-450e-9098-770f9831d81b.png)

Reply:
cool 👍  well then just ignore the comment 😄